Exception due to " class mismatch: java.lang.Long != java.math.BigInteger "

### Description

Nightly regression test **SmartContractOps-NIReconnect-14-21m** failed for multiple days

Slack report 

https://swirldslabs.slack.com/archives/C03E8SA5UF9/p1669711647517409

And there are following client error in logs


```
2022-11-29 08:24:40.553 ERROR  61   CustomSpecAssert - Operation 'HapiContractCallLocal{payer=GENESIS, node=0.0.3, contract=BalanceLookup, abi={"outputs":[{"name":"","internalType":"uint256","type":"uint256"}],"constant":true,"payable":false,"inputs":[{"name":"accountNum","internalType":"uint64","type":"uint64"}],"name":"lookup","stateMutability":"view","type":"function"}, params=[1002]}' :: tuple index 0: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
2022-11-29 08:24:40.553 WARN   234  HapiSpecOperation - 'RunMixedSmartContractOps' - HapiContractCallLocal{payer=GENESIS, node=0.0.3, contract=BalanceLookup, abi={"outputs":[{"name":"","internalType":"uint256","type":"uint256"}],"constant":true,"payable":false,"inputs":[{"name":"accountNum","internalType":"uint64","type":"uint64"}],"name":"lookup","stateMutability":"view","type":"function"}, params=[1002]} failed tuple index 0: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)!
Exception in thread "thread1" java.lang.IllegalStateException: java.lang.IllegalArgumentException: tuple index 0: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.handleExec(CustomSpecAssert.java:62)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.executeHederaOps(CustomSpecAssert.java:46)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:39)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:67)
2022-11-29 08:24:40.554 ERROR  61   CustomSpecAssert - Operation 'HapiContractCallLocal{payer=GENESIS, node=0.0.3, contract=BalanceLookup, abi={"outputs":[{"name":"","internalType":"uint256","type":"uint256"}],"constant":true,"payable":false,"inputs":[{"name":"accountNum","internalType":"uint64","type":"uint64"}],"name":"lookup","stateMutability":"view","type":"function"}, params=[1002]}' :: tuple index 0: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:245)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$22(RunLoadTest.java:211)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: tuple index 0: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
	at com.esaulpaugh.headlong.abi.TupleType.countBytes(TupleType.java:131)
	at com.esaulpaugh.headlong.abi.TupleType.countBytes(TupleType.java:120)
	at com.esaulpaugh.headlong.abi.TupleType.validate(TupleType.java:138)
	at com.esaulpaugh.headlong.abi.Function.validatedCallLength(Function.java:192)
	at com.esaulpaugh.headlong.abi.Function.encodeCall(Function.java:204)
	at com.esaulpaugh.headlong.abi.Function.encodeCallWithArgs(Function.java:200)
	at com.hedera.services.bdd.spec.transactions.contract.HapiParserUtil.encodeParameters(HapiParserUtil.java:40)
	at com.hedera.services.bdd.spec.transactions.contract.HapiParserUtil.encodeParametersForCall(HapiParserUtil.java:29)
	at com.hedera.services.bdd.spec.queries.contract.HapiContractCallLocal.getContractCallLocal(HapiContractCallLocal.java:188)
	at com.hedera.services.bdd.spec.queries.contract.HapiContractCallLocal.lookupCostWith(HapiContractCallLocal.java:169)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.timedCostLookupWith(HapiQueryOp.java:321)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.fittedPayment(HapiQueryOp.java:262)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.submitOp(HapiQueryOp.java:154)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:210)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.handleExec(CustomSpecAssert.java:59)
	... 6 more
Caused by: java.lang.IllegalArgumentException: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
	at com.esaulpaugh.headlong.abi.ABIType.mismatchErr(ABIType.java:128)
	at com.esaulpaugh.headlong.abi.ABIType.validateClass(ABIType.java:120)
	at com.esaulpaugh.headlong.abi.ABIType._validate(ABIType.java:111)
	at com.esaulpaugh.headlong.abi.TupleType.validateObject(TupleType.java:145)
	at com.esaulpaugh.headlong.abi.TupleType.lambda$validate$3(TupleType.java:138)
	at com.esaulpaugh.headlong.abi.TupleType.countBytes(TupleType.java:127)
	... 20 more
Exception in thread "thread0" java.lang.IllegalStateException: java.lang.IllegalArgumentException: tuple index 0: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.handleExec(CustomSpecAssert.java:62)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.executeHederaOps(CustomSpecAssert.java:46)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:39)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.allRunFor(CustomSpecAssert.java:67)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.testRun(RunLoadTest.java:245)
	at com.hedera.services.bdd.spec.utilops.RunLoadTest.lambda$threadMode$22(RunLoadTest.java:211)
	at java.base/java.lang.Thread.run(Thread.java:833)
Caused by: java.lang.IllegalArgumentException: tuple index 0: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
	at com.esaulpaugh.headlong.abi.TupleType.countBytes(TupleType.java:131)
	at com.esaulpaugh.headlong.abi.TupleType.countBytes(TupleType.java:120)
	at com.esaulpaugh.headlong.abi.TupleType.validate(TupleType.java:138)
	at com.esaulpaugh.headlong.abi.Function.validatedCallLength(Function.java:192)
	at com.esaulpaugh.headlong.abi.Function.encodeCall(Function.java:204)
	at com.esaulpaugh.headlong.abi.Function.encodeCallWithArgs(Function.java:200)
	at com.hedera.services.bdd.spec.transactions.contract.HapiParserUtil.encodeParameters(HapiParserUtil.java:40)
	at com.hedera.services.bdd.spec.transactions.contract.HapiParserUtil.encodeParametersForCall(HapiParserUtil.java:29)
	at com.hedera.services.bdd.spec.queries.contract.HapiContractCallLocal.getContractCallLocal(HapiContractCallLocal.java:188)
	at com.hedera.services.bdd.spec.queries.contract.HapiContractCallLocal.lookupCostWith(HapiContractCallLocal.java:169)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.timedCostLookupWith(HapiQueryOp.java:321)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.fittedPayment(HapiQueryOp.java:262)
	at com.hedera.services.bdd.spec.queries.HapiQueryOp.submitOp(HapiQueryOp.java:154)
	at com.hedera.services.bdd.spec.HapiSpecOperation.execFor(HapiSpecOperation.java:210)
	at com.hedera.services.bdd.spec.utilops.CustomSpecAssert.handleExec(CustomSpecAssert.java:59)
	... 6 more
Caused by: java.lang.IllegalArgumentException: class mismatch: java.lang.Long != java.math.BigInteger (uint64 requires BigInteger but found Long)
	at com.esaulpaugh.headlong.abi.ABIType.mismatchErr(ABIType.java:128)
	at com.esaulpaugh.headlong.abi.ABIType.validateClass(ABIType.java:120)
	at com.esaulpaugh.headlong.abi.ABIType._validate(ABIType.java:111)
	at com.esaulpaugh.headlong.abi.TupleType.validateObject(TupleType.java:145)
	at com.esaulpaugh.headlong.abi.TupleType.lambda$validate$3(TupleType.java:138)
	at com.esaulpaugh.headlong.abi.TupleType.countBytes(TupleType.java:127)
	... 20 more

```
[hedera-services-automation_develop_results_6N_1C_Reconnect_20221129-072525-GCP-Daily-Comp-Reconnect-6N-1C_SmartContractOps-NIReconnect-14-21m_node0000-TestClient_hapi-client-combined.log](https://github.com/hashgraph/hedera-services/files/10117033/hedera-services-automation_develop_results_6N_1C_Reconnect_20221129-072525-GCP-Daily-Comp-Reconnect-6N-1C_SmartContractOps-NIReconnect-14-21m_node0000-TestClient_hapi-client-combined.log)


### Steps to reproduce

Run nightly regression test **GCP-Daily-Comp-Reconnect-6N-1C**

### Additional context

_No response_

### Hedera network

other

### Version

v1.0

### Operating system

Linux

--- a/test-clients/src/main/java/com/hedera/services/bdd/suites/perf/contract/MixedSmartContractOpsLoadTest.java
+++ b/test-clients/src/main/java/com/hedera/services/bdd/suites/perf/contract/MixedSmartContractOpsLoadTest.java
@@ -34,6 +34,7 @@ import com.hedera.services.bdd.spec.HapiSpecOperation;
 import com.hedera.services.bdd.spec.HapiSpecSetup;
 import com.hedera.services.bdd.spec.utilops.LoadTest;
 import com.hedera.services.bdd.suites.perf.PerfTestLoadSettings;
+import java.math.BigInteger;
 import java.util.List;
 import java.util.concurrent.atomic.AtomicInteger;
 import java.util.function.Supplier;
@@ -68,7 +69,7 @@ public class MixedSmartContractOpsLoadTest extends LoadTest {
         final String PAYABLE_CONTRACT = "PayReceivable";
         final String LOOKUP_CONTRACT = "BalanceLookup";
         final String CIVILIAN_ACCOUNT = "civilian";
-        final int depositAmount = 1;
+        final BigInteger depositAmount = BigInteger.ONE;
 
         Supplier<HapiSpecOperation[]> mixedOpsBurst =
                 () ->
@@ -91,13 +92,15 @@ public class MixedSmartContractOpsLoadTest extends LoadTest {
                                             "lookup",
                                             spec ->
                                                     new Object[] {
-                                                        spec.registry()
-                                                                .getAccountID(CIVILIAN_ACCOUNT)
-                                                                .getAccountNum()
+                                                        BigInteger.valueOf(
+                                                                spec.registry()
+                                                                        .getAccountID(
+                                                                                CIVILIAN_ACCOUNT)
+                                                                        .getAccountNum())
                                                     })
                                     .payingWith(GENESIS),
                             contractCall(PAYABLE_CONTRACT, "deposit", depositAmount)
-                                    .sending(depositAmount)
+                                    .sending(depositAmount.longValueExact())
                                     .suppressStats(true)
                                     .deferStatusResolution()
                         };

